functions/AuthUtils.js

```diff
@@ -0,0 +1,186 @@
+const { Issuer } = require('openid-client');
+const { generators } = require('openid-client');
+const jwt = require('jsonwebtoken');
+const cookie = require('cookie');
+
+const getOpenIDClient = async () => {
+    const issuer = await Issuer.discover(`https://${process.env.AUTH0_DOMAIN}`);
```

Comment:
TODO: Hard-code this to see if it makes a noticeable difference in load time and not a big impact on DX.